### DISCUSSION TITLE
Undoing getting extra lives in obfuscated jar

### DISCUSSION THREAD

**Discussion by tcrl** [CastleLab/COMP3021-2021Fall-PA1-Student-Version#23] [Q&A] [answered]
I took the example in the Undo section of the readme (and added a few cells to the right so the game doesn't end on me immediately) and ran it in the obfuscated jar. However, the number of lives kept increasing (1->2->3) instead of going back down (1->2->1). Is this supposed to happen? 

This is what I got with the obfuscated jar:

<img width="467" alt="Screenshot 2021-09-28 at 10 21 07 PM" src="https://user-images.githubusercontent.com/90368282/135106195-e5420bd5-ac77-45f9-8562-513b535bcf7c.png">

Thanks!

**Comment by Derppening** [comment, marked as answer]
Are you using the new obfuscated JAR or the old one? 29d891b (which was updated a few days ago) contained a fix for this issue.
